### ISSUE TITLE
AdvancedDataGridDemo for newer Avalonia versions

### ISSUE THREAD

**Issue by GitClickOk** [npolyak/NP.Avalonia.Demos#5]
I am creating a new avalonia  app  with:
`dotnet new avalonia.mvvm`
and then I am trying to make changes to the template to add the Advanced DataGrid changes, but it does not work. when I execute 
`dotnet run`
nothing happens, I didn't receive even errors...
I think it can be because of breaking changes in styles (https://github.com/AvaloniaUI/Avalonia/wiki/Breaking-Changes#default-theme-was-renamed-to-simple) where says "_StyleInclude with old relative path won't work anymore._"

1. What can I do to this work with new Avalonia version?
2. I'm using Avalonia 0.10.18, will this work with the Avalonia vNext (version 11)?

**Comment by npolyak** [comment]
No, unfortunately the older avalonia versions won't work with the newest. I plan to update Avalonia versions for my projects soon. 

To use the newest Simple style you should add the following XAML to your App.axaml file:

    <Application.Styles>
        <SimpleTheme Mode="Light"/>
    </Application.Styles>
